Decred Wallet/Features/Navigation Menu/NavigationMenuViewController.swift

```diff
@@ -54,6 +54,7 @@ class NavigationMenuViewController: UIViewController {
         if BuildConfig.IsTestNet {
             decredHeaderLogo?.image = UIImage(named: "logo-testnet")
         }
+        decredHeaderLogo.backgroundColor = UIColor.appColors.offWhite
```

Comment:
I noticed that the image is assigned a background color in Inspector Builder. Is this background color assignment necessary?

Reply:
Not necessary anymore, deleted